docker/errors.go

```diff
@@ -10,9 +10,6 @@ import (
 )
 
 var (
-	// ErrV1NotSupported is returned when we're trying to talk to a
-	// docker V1 registry.
-	ErrV1NotSupported = errors.New("can't talk to a V1 container registry")
```

Comment:
This is a public API and can’t be removed, even if it will never match anything.

Adding a `Deprecated:` comment would be useful.